modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/InviteTeamMembersForm/index.js

```diff
@@ -369,28 +374,38 @@ const InviteTeamMembersPage = ({
 									<h5 className="text-neutral-7">
 										{`${
 											projectHasSLAGoldPlatinum
-												? 'Support Seats'
-												: `${ROLE_TYPES.admin.name} roles`
-										}   available: ${availableAdminsRoles} of ${
-											project.maxRequestors
-										}`}
+												? i18n.translate(
+														'support-seats'
+												  )
+												: `${
+														ROLE_TYPES.admin.name
+												  } ${i18n.translate('roles')}`
```

Comment:
Maybe you can add the administrator name, like: create a key as administrator-roles